src/var/flat.js

```diff
@@ -0,0 +1,18 @@
+define( [
+	"./arr",
+	"./concat"
+], function( arr, concat ) {
+
+"use strict";
+
+// Support: IE, chrome < 69, firefox < 62, opera < 56, safari < 12
```

Comment:
The format of support comments should follow the syntax described in:
https://github.com/jquery/contribute.jquery.org/issues/95#issuecomment-69379197
(examples in the table down the comment) with my modification:
https://github.com/jquery/contribute.jquery.org/issues/95#issuecomment-448998379

In particular: browser names start with an uppercase letter, please mind the spacing, each browsers needs a version range specified & each of them needs to end with `only` or with `+`. Also, there's no need to include Chrome, Firefox & Opera here as the issue there is only in versions we don't support. You should add Edge, though.

Reply:
IE & Edge doesn't support Array#flat at all.
Could you suggest its format, because I don't know how to define version range while the range is undefined (opened)!

Reply:
Did you have a look at the second comment I linked above? It provides a few examples. Basically, for Edge I'd expect `Edge 12 - 18+` or `Edge 18+`, for IE: `IE 11+`.